src/components/ProductInfo/ProductInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+
+import cn from 'classnames'
+
+import { useToggle } from 'hooks/toggleHook'
+import { IProduct } from 'interface/productInterface'
+import { useGetProductQuery } from 'store/services/product'
+
+import { MoreInformation } from 'components/MoreInformation'
+
+import { textAboutMinHeight } from './constants/textAboutMinHeight'
+import { ProductCharacteristics } from './ProductCharacteristics'
+import { ProductPictures } from './ProductPictures'
+import { ProductsByCategory } from './ProductsByCategory'
+import { ProductsByType } from './ProductsByType'
+
+import styles from './ProductInfo.module.scss'
+
+type Props = {
+  id: string
+}
+
+export const ProductInfo: FC<Props> = ({ id }) => {
+  const { data, isSuccess } = useGetProductQuery(id)
+
+  const product = isSuccess ? data : {} as IProduct
+
+  const { isOpen: isOpenTextAbout, onToggle: toggleTextAbout } = useToggle(true)
+
+  return (
+    <div className={styles.wrapper}>
+      {isSuccess &&
```

Comment:
if `isSuccess === false` we will render blank `div` i guess it's nut correct

Reply:
added loader